cibuild.sh

```diff
@@ -299,6 +299,12 @@ if [[ $BOOTSTRAP_ONLY = true ]]; then
     exit $?
 fi
 
+# Microsoft.Net.Compilers package is available now, so we can use the latest csc.exe
+if [ "$host" = "Mono" ]; then
+        CSC_EXE=`find $PACKAGES_DIR/microsoft.net.compilers -name csc.exe`
```

Comment:
Hm good point. And once we have the mono CI enabled, breakages (referenced version change) should show up!